rubocop.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |s|
     'bug_tracker_uri' => 'https://github.com/bbatsov/rubocop/issues'
   }
 
+  s.add_runtime_dependency('jaro_winkler', '~> 1.4.0')
```

Comment:
Why? [There is newer `1.5.0` version](https://rubygems.org/gems/jaro_winkler/versions/1.5.0).

Reply:
That's a mistake. It should be 1.5.0. I'll fix that.

Reply:
OK, thank you!

Reply:
The build failed on JRuby, see #5959, so we're keeping 1.4.0.

Reply:
We should ask for JRuby release, I see no reasons not to do this release: https://github.com/tonytonyjan/jaro_winkler/issues/19

Reply:
hold on, I'm working on it. 💻

Reply:
jaro_winkler v1.5.1 has just been released for supporting platforms other than MRI. Please use this version instead of v1.5.0. I've tested on jruby 9.1.17.0 and it worked well.

Let me know if there is anything else I could help. 🍻

Reply:
Thank you, @tonytonyjan!